packages/celotool/src/e2e-tests/transfer_tests.ts

```diff
@@ -829,4 +846,74 @@ describe('Transfer tests', function(this: any) {
       })
     }
   })
+
+  describe('Transfers Frozen >', () => {
+    before(restartWithCleanNodes)
+
+    for (const syncMode of syncModes) {
+      describe(`${syncMode} Node >`, () => {
+        before(`start geth on sync: ${syncMode}`, async () => {
+          try {
+            await startSyncNode(syncMode)
+          } catch (err) {
+            console.debug('some error', err)
```

Comment:
Are these errors always safe to ignore?

Reply:
Good call. I copied that part from 'Transfer with changed intrinsic gas cost >' block of tests, but I think after looking at it again it was probably left in to catch errors from the call to setIntrinsicGas(..) which I'm not using. I think taking out the catch here is probably the right thing. Thanks!